org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/folding/LSPFoldingReconcilingStrategy.java

```diff
@@ -63,6 +64,9 @@ public class LSPFoldingReconcilingStrategy
 
 	private static final Annotation[] NO_ANNOTATIONS = new Annotation[0];
 
+	private static final Pattern LICENSE_KEYWORDS = Pattern
+			.compile("(?i)(copyright|licensed under|all rights reserved|SPDX-License-Identifier)"); //$NON-NLS-1$
```

Comment:
Is this kind of folding not something that should be determined by de server, which can know more about the language?

Reply:
Yes, probably, but the LSP does specify such information at the moment; so we cannot use anything like it so far.

Reply:
This was discussed/denied here https://github.com/microsoft/language-server-protocol/pull/1819

Reply:
thanks